src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.BuildOrder.targets

```diff
@@ -62,7 +62,6 @@ properties that determine build ordering.
     <_PrepareBuildApkDependsOnTargets>
       _SetLatestTargetFrameworkVersion;
       _GetLibraryImports;
-      _RemoveRegisterAttribute;
```

Comment:
Is this change needed? I didn't see anything in here that is trying to reorder MSBuild targets?

Maybe it's from before, trying to workaround the bug in https://github.com/dotnet/android/pull/9452?

Reply:
It's possible, I don't remember exactly which test was affected by this change.  I'll revert this bit and we'll see what (if anything) breaks.

Reply:
Did this revert work?